dvc/command/data_sync.py

```diff
@@ -38,13 +40,17 @@ def do_run(self, target=None):
         except Exception:
             logger.error("failed to pull data from the cloud")
             return 1
+        if processed_files_count == 0:
```

Comment:
Let's create `check_count`(or maybe some better name) staticmethod in CmdDataBase above and use it in every class down below instead of copy-pasting two lines :)

Reply:
https://github.com/iterative/dvc/pull/1620/commits/d05e01a86e4859c0c560e53a4314f1f6b95f05fa